config.toml

```diff
@@ -9,6 +9,7 @@ description = "Guidelines for building sustainable free and open source communit
 disableBreadcrumb = false
 themeVariant = "sfosc"
 editURL = "https://github.com/sfosc/sfosc/edit/master/content/"
+isPreview = false
```

Comment:
If we are checking for isPreview to be true, wouldn't it also work to leave it out here? Or is the idea to be explicit that this file is not for preview?

```
{{if .Site.Params.IsPreview}}
        <div id="preview-banner">
          This is an early preview of the SFOSC site.
          The current release is available on <a href="https://sfosc.org/">sfosc.org</a>.
        </div>
        {{end}}
```
Also could you give me a high level overview of when, and how this preview is generated?

Reply:
Every page will have the banner when this variable is set. Using `.Site` means it can only be set in the global configuration. There's no need to handle unset, because I've changed both versions of the config to be explicit. But yes I think unsetting the variable would have the same result.

Currently the when, is when I manually run the `deploy-preview.sh`. I would like to change this to be CI based instead as the next step. I've managed to do this for sourcecred, so a similar setup would be what I'd do for preview.

Reply:
ah okay, this makes sense! You probably have something in mind already, but if not, circleci does a good job with static previews (artifacts).